examples/Storage/Storage.Tests.ps1

```diff
@@ -116,3 +176,8 @@ Describe 'Verify Storage Container' {
     $result | Should -BeDeployed
   }
 }
+
+AfterAll {
+  Get-Module Az-InfrastructureTesting | Remove-Module
+  Get-Module BenchPress.Azure | Remove-Module
```

Comment:
Do we need `Get-Module BenchPress.Azure | Remove-Module` if we never import it at the top?

Reply:
I meant to mention this at the design session, but I forgot.

In the documentation we give two ways to use the example, by importing after installing from the PS Gallery, and by importing from the local project after building. By adding these two lines, in this manner, it ensures that regardless of how the user consumes the example it will properly remove the module at the end.

Reply:
The thing that I *really* don't like about it, is that they are named differently.

Reply:
That makes sense. Maybe add a line to the `getting-started.md` about it?

Reply:
That's a good idea. I'll add it!